etc/building-locally.md

```diff
@@ -0,0 +1,241 @@
+    ```
+    export auth_health=https://localhost:9443/health
+    export inventory_url=http://localhost:9081/inventory/rest/inventory/stock
+    export inventory_health=http://localhost:9081/health
+    ```
+    
+2. To enable authentication, the [Auth MicroService](https://github.com/ibm-cloud-architecture/refarch-cloudnative-auth/tree/microprofile) 
+must be running and the keystore must be set up. Please refer to the link for further instructions.
+
+1. Start your server.
+
+    ```
+    mvn liberty:start-server -DtestServerHttpPort=9083 -DtestServerHttpsPort=9446
+    ```
+
+    You will see something similar to the below messages.
+
+    ```
+    [INFO] Starting server defaultServer.
+    [INFO] Server defaultServer started with process ID 14781.
+    [INFO] Waiting up to 30 seconds for server confirmation:  CWWKF0011I to be found in /Users/user@ibm.com/BlueCompute/refarch-cloudnative-micro-orders/orders/target/liberty/wlp/usr/servers/defaultServer/logs/messages.log
+    [INFO] CWWKM2010I: Searching for CWWKF0011I in /Users/user@ibm.com/BlueCompute/refarch-cloudnative-micro-orders/orders/target/liberty/wlp/usr/servers/defaultServer/logs/messages.log. This search will timeout after 30 seconds.
+    [INFO] CWWKM2015I: Match number: 1 is [18/7/18 14:42:32:751 CDT] 0000001a com.ibm.ws.kernel.feature.internal.FeatureManager            A CWWKF0011I: The server defaultServer is ready to run a smarter planet..
+    [INFO] ------------------------------------------------------------------------
+    [INFO] BUILD SUCCESS
+    [INFO] ------------------------------------------------------------------------
+    [INFO] Total time: 9.889 s
+    [INFO] Finished at: 2018-07-18T14:42:32-05:00
+    [INFO] Final Memory: 14M/309M
+    [INFO] ------------------------------------------------------------------------
+    ```
+    
+    From here, you may access the Orders' endpoints, like `/health` or `/openapi/ui`. 
+    To hit secure REST endpoints, like `/orders/rest/orders` to retrieve our list of orders, we need to 
+    authenticate our calls appropriately. Please refer [here](https://github.com/ibm-cloud-architecture/refarch-cloudnative-auth/blob/microprofile/building-locally.md) 
+    for further details.
+
+2. Retrieve the JWT from the Auth Service to authorize secure REST calls:
+    ```
+    curl -k -d 'grant_type=password&client_id=bluecomputeweb&client_secret=bluecomputewebs3cret&username=user&password=password&scope=openid' https://localhost:9443/oidc/endpoint/OP/token
+    ```
+    You should see something similar to this:
+    ![jwt_example](jwt_example.png)
+    
+3. Setup your [keystore](about:blank) <-- TODO (Waiting for JJ's changes)
+    
+2. Validate the orders service. Insert your given JWT and you should get a list of items.
+    ```
+    curl -k --request GET \
+      --url https://localhost:9446/orders/rest/orders \
+      --header 'Authorization: Bearer <Insert Token Here>
+      --header 'Content-Type: application/json'
+    ```
```

Comment:
Add post and then get so that we can see the existing orders. Without POST it displays []

curl -k -H "Content-Type: application/json" -H "Authorization: Bearer <Insert Token Here>" -X POST -d '{"itemId":13401, "count":1}' https://localhost:9446/orders/rest/orders